README.md

```diff
@@ -101,26 +107,111 @@ export function loadConfig(oidcConfigService: OidcConfigService) {
 export class AppModule {
     constructor(private oidcSecurityService: OidcSecurityService, private oidcConfigService: OidcConfigService) {
         this.oidcConfigService.onConfigurationLoaded.subscribe(() => {
-            const config = new OpenIDImplicitFlowConfiguration();
-            config.stsServer = 'https://your_secure_token_service_url';
-            config.redirect_url = 'https://localhost:4200';
-            config.client_id = 'angular_code_client';
-            config.scope = 'openid profile email';
-            config.response_type = 'code';
+          const config = new OpenIDImplicitFlowConfiguration();
+          //merge configuration loaded from assets/auth.clientConfiguration.json
+          Object.assign(implicitFlowConfig, this.oidcConfigService.clientConfiguration);
```

Comment:
Shouldn't this be `Object.assign(config, this.oidcConfigService.clientConfiguration);`?